api/mesh/v1alpha1/metrics.proto

```diff
@@ -90,3 +93,25 @@ message PrometheusEnvoyConfig {
   // least once). If nil, then it is treated as false.
   google.protobuf.BoolValue usedOnly = 2;
 }
+
+// PrometheusEnvoyConfig defines Tls configuration for Prometheus listener.
+message PrometheusTlsConfig {
+  enum Mode {
+    // This is a substitute of `skipMTLS` set to false. In this situation,
+    // control-plane delivers certificates to the prometheus listener.
+    activeMTLSBackend = 0;
+    // In this way user is resposible for providing certificates to dataplanes.
+    // Path for the certificte and the key needs to be provided to the dataplane
+    // by environments variables:
+    // * KUMA_DATAPLANE_METRICS_CERT_PATH
+    // * KUMA_DATAPLANE_METRICS_KEY_PATH
+    delegated = 1;
+  }
+  // If true then endpoints for scraping metrics will require TLS, when false
+  // there if no tls configured. If nil, then it is treated as true.
+  google.protobuf.BoolValue enabled = 1;
```

Comment:
Instead of having this should we have a mode that is `disabled` and avoid having 2 params?

Reply:
Seems good, `activeMTLSBackend` could be the default.